lib/helpers/client-formatter.js

```diff
@@ -20,12 +20,24 @@ module.exports = class ClientFormatter {
 		return this._statusActive;
 	}
 
-	static format(code) {
-		return {
+	static format({ code, ...client }, additionalFields) {
+
+		const formattedClient = {
 			code,
 			databases: this.prepareDatabases(code),
 			status: this.statusActive
 		};
+
+		if(!additionalFields)
+			return formattedClient;
+
+		Object.entries(client).forEach(([key, value]) => {
+
+			if(additionalFields.includes(key))
+				formattedClient[key] = value;
+		});
```

Comment:
Propongo recorrer los campos adicionales.. ya que es probable que sean menos que recorrer los campos del client
```suggestion
		additionalFields.forEach(key => {
			if(client[key])
				formattedClient[key] = client[key];
		});
```